flutter_theoplayer_sdk/flutter_theoplayer_sdk_ios/ios/theoplayer_ios.podspec

```diff
@@ -22,6 +22,7 @@ THEOplayer Flutter iOS SDK
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
   s.swift_version = '5.0'
 
-  s.dependency 'THEOplayerSDK-core', '8.11.0'
+  s.dependency 'THEOplayerSDK-core', '8.13.0-theolive1'
+  s.dependency 'THEOplayer-Integration-THEOlive', '8.13.0-theolive1'
```

Comment:
we rely on 8.13.0-theolive1 pre-release for now, once THEOplayer 8.13.0 is out, we can update.